### PR TITLE
refactor: clean up types and imports

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by jkwlui** [comment]
@stephenplusplus are you able to look into why system tests are failing here?

**Comment by stephenplusplus** [comment]
I can't reproduce locally. Let's try a rebuild!

**Comment by stephenplusplus** [comment]
Rebuild still failed.

@jkwlui-- the error message, "Error: 3 INVALID_ARGUMENT: A monitored resource must be specified for each log entry", could relate to the logic where we find the most appropriate resource to go alongside a log entry dependent on the environment the code is in. That being said, I'm not sure how to simulate the Kokoro environment. Any ideas?

**Comment by JustinBeckwith** [comment]
Can you try something?  If the grpc-js env var is set, just process.exit?  I wonder if this is a race condition with system tests.

**Comment by stephenplusplus** [comment]
I'm not 100% on what that env var is, but I went with this: https://github.com/googleapis/nodejs-logging/pull/409/commits/53691a1676236e48c3dc2ec59ef002cd60a1d8c9

Looks like Kokoro is having issues, though, so the suspense continues!

**Comment by JustinBeckwith** [comment]
@jkwlui is on the other issues

File: src/index.ts [hunk 1]
```diff
@@ -340,10 +339,10 @@ class Logging {
       name: string, config: CreateSinkRequest,
       callback?: CreateSinkCallback): Promise<[Sink, LogSink]>|void {
     const self = this;
-    if (!is.string(name)) {
+    if (typeof name !== 'string') {
```
Comment (ofrobots) [inline src/index.ts]: Would this be considered semver major?

[rant] Removal of `is` as a dependency doesn't fit into the spirit of this PR anyway. This only makes the review process more complex. Refactor PRs should be easy to review.

Open a separate PR for this change.
[/rant]

Comment (JustinBeckwith) [reply src/index.ts]: Happy to :) So question tho - would you prefer a total drop of `is`, a move to the other `is`, or keeping this as `is` 🥁

Comment (ofrobots) [reply src/index.ts]: No objections to dropping `is` as semver-major. If the argument is that it is not breaking, it needs to be made.

**Pull request merged**